idearium-lib/lib/util/index.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const find = require('./find');
+const has = require('./has');
+const is = require('./is');
+const to = require('./to');
+// Destructure to remove deprecated and conflicting apis.
+const { debuglog, deprecate, format, inherits, inspect } = require('util');
+
+// Merge all the functions and the node built in util library.
+const utilLib = Object.assign(
```

Comment:
I don't think you need the `utilLib` variable. Just export the result directly. Unnecessary variables is something that Eric Elliot writes about too.